internal/zinc-compile-core/src/test/scala/sbt/internal/inc/javac/JavaCompilerSpec.scala

```diff
@@ -60,7 +66,9 @@ class JavaCompilerSpec extends UnitSpec with DiagrammedAssertions {
     assert(result)
     val classfile = new File(out, "good.class")
     assert(classfile.exists)
-    assert(classfileManager.generatedClasses == (if (forked) Set() else Set(classfile)))
+    assert(
```

Comment:
This is a hack 😛I presume you're doing it because in macOS temporary files get a public symlink to a private symlink and here you are getting the dealiased paths for the generated class files. Can we instead dealias `class file`?

Reply:
ok.

Reply:
https://github.com/sbt/zinc/pull/640/commits/23485a8fa5ebc208805727c1121f9126874022dd

```scala
  def dealiasSymlinks(xs: HashSet[File]): HashSet[String] =
    xs map { x =>
      x.getCanonicalPath
    }
```